### PR TITLE
Add books

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
## Summary

Add list of books.

## Screenshots

LIST OF BOOKS

## Trello Card

https://trello.com/c/ro0n7jlq/68-book-list-auth-header

### CODE REVIEW

File: react/wbooks/src/app/components/PublicRoute/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ function PublicRoute({ component: Component, redirect, ...rest }) {
     <Route
       {...rest}
       render={props =>
-        localStorage.getItem('token_id') ? <Redirect to={redirect} /> : <Component {...props} />
+        localStorage.getItem('tokenId') ? <Redirect to={redirect} /> : <Component {...props} />
```
Comment (LucasZibell) [inline react/wbooks/src/app/components/PublicRoute/index.js]: You can use only 1 component `Route` and pass a prop `private`. If it is private you negate this condition. And you end up using the same component for the two routes

File: react/wbooks/src/app/index.js [hunk 2]
```diff
@@ -11,11 +10,9 @@ import PublicRoute from './components/PublicRoute';
...
   return (
     <Router>
-      <div className={styles.container}>
-        <PublicRoute exact path="/" component={Login} redirect="/home" />
-        <PrivateRoute exact path="/home" component={Home} redirect="/" />
-        <PublicRoute path="/sign_up" component={SignUp} redirect="/home" />
-      </div>
+      <PublicRoute exact path="/" component={Login} redirect="/home" />
+      <PrivateRoute exact path="/home" component={Home} redirect="/" />
+      <PublicRoute path="/signUp" component={SignUp} redirect="/home" />
```
Comment (LucasZibell) [inline react/wbooks/src/app/index.js]: specify the redirect only when neccesary. And keep '/home' and '/' as the default redirect on the `Route` component

File: react/wbooks/src/app/screens/Home/components/Book/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+function Book({ title, author }) {
+  return useMemo(
+    () => (
+      <div className={styles.bookCover}>
+        <img className={styles.bookImage} src={logoBook} alt="Wolox" />
+        <h3 className={styles.title}>{title}</h3>
+        <p className={styles.author}>{author}</p>
+      </div>
+    ),
+    [author, title]
```
Comment (LucasZibell) [inline react/wbooks/src/app/screens/Home/components/Book/index.js]: Hey, don't use useMemo for renders. React already renders this only when title and author change. The useMemo hook should be use for complicated calculations before a render

File: react/wbooks/src/app/screens/Home/index.js [hunk 4]
```diff
@@ -1,23 +1,49 @@
...
+import { getBooks } from '../../../services/BookServices';
 
+import actionCreators from './reducer/actions';
+import reducer, { initialState } from './reducer/reducer';
 import styles from './styles.module.scss';
+import Book from './components/Book';
 
 const clearTokens = () => {
-  localStorage.setItem('token_id', '');
+  localStorage.setItem('tokenId', '');
```
Comment (LucasZibell) [inline react/wbooks/src/app/screens/Home/index.js]: arrow function

**Review by LucasZibell — CHANGES_REQUESTED** [review]

File: react/wbooks/src/app/index.js [hunk 5]
```diff
@@ -11,11 +10,9 @@ import PublicRoute from './components/PublicRoute';
 function App() {
   return (
     <Router>
-      <div className={styles.container}>
-        <PublicRoute exact path="/" component={Login} redirect="/home" />
-        <PrivateRoute exact path="/home" component={Home} redirect="/" />
-        <PublicRoute path="/sign_up" component={SignUp} redirect="/home" />
-      </div>
+      <PublicRoute exact path="/" component={Login} />
+      <PrivateRoute exact path="/home" component={Home} />
```
Comment (mariadelacamara) [inline react/wbooks/src/app/index.js:14]: why do you write `exact` path here?

**Review by LucasZibell — APPROVED** [review]

**Pull request merged**